src/momentRe.re

```diff
@@ -131,6 +131,14 @@ external momentWithTimestampMS : float => Moment.t = "moment" [@@bs.module];
 
 external momentWithComponents : list int => Moment.t = "moment" [@@bs.module];
 
+external diff :
+  Moment.t =>
+  Moment.t =>
+  [ | `years | `quarters | `months | `weeks | `days | `hours | `minutes | `seconds | `milliseconds]
+  [@bs.string] =>
+  int =
```

Comment:
maybe larger than int?

Reply:
You mean like Big_int? I don't think that module currently works in Reason. Were you thinking of another type? It was pointed to me that this library is javascript so we can't have more than 53 bit numbers. Do you know if there is a type for those in Reason?

Reply:
so maybe use float64 or int64?

Reply:
float is 64-bit. That's why it's used in [`Js.Date`](https://bucklescript.github.io/bucklescript/api/Js.Date.html)

Reply:
I changed the return type to `float`in 308f200